libs/utils/trace.py

```diff
@@ -73,7 +73,7 @@ class Trace(object):
     :type plots_prefix: str
     """
 
-    def __init__(self, platform, data_dir, events,
+    def __init__(self, platform, data_dir, events=None,
```

Comment:
:heart_eyes: another step into the direction to make `Trace()` super simple to build, thanks! ;-)

I still have to provide a patch to make `platform` optional as well... will do soon.